pwiz_tools/Skyline/Model/Lib/Library.cs

```diff
@@ -2508,6 +2511,100 @@ public string FileName
         public double? RetentionTime { get; set; }
         public IonMobilityAndCCS IonMobilityInfo { get; private set; }
         public string Protein { get; private set; } // Also used as Molecule List Name for small molecules
+

```

Comment:
Avoid adding this many consecutive empty lines. Most of our code using single empty line spacing.